### PR TITLE
Breakpoint contextmenu: add "Add/Edit breakpoint condition"

### PR DESCRIPTION
**Pull request by nyrosmith**
Associated Issue: NONE

@jasonLaster suggested this feature on slack

### Summary of Changes

* renamed contextmenu entry 
* jump to breakpoint location

conditional

### CODE REVIEW

File: src/components/SecondaryPanes/Breakpoints.js [hunk 1]
```diff
@@ -248,11 +248,16 @@ class Breakpoints extends PureComponent {
...
     };
 
-    const editCondition = {
+    const addOrEditCondition = {
       id: "node-menu-edit-condition",
-      label: editConditionLabel,
-      accesskey: editConditionKey,
-      click: () => toggleConditionalBreakpointPanel(breakpoint.location.line)
+      label: addOrEditConditionLabel,
+      accesskey: addOrEditConditionKey,
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Breakpoints.js]: I think we should have two separate labels "Add Condition", "Edit Condition" that we toggle based on whether there is a condition. We do the same thing when you right click on the gutter.

File: src/components/SecondaryPanes/Breakpoints.js [hunk 2]
```diff
@@ -248,11 +248,16 @@ class Breakpoints extends PureComponent {
...
+      click: () => {
+        const sourceId = breakpoint.location.sourceId;
+        const line = breakpoint.location.line;
+        this.props.selectSource(sourceId, { line });
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Breakpoints.js]: yes! good catch. Perhaps we move this invokation to this action: `toggleConditionalBreakpointPanel`

**Comment by jasonLaster** [comment]
screen shot 2017-09-20 at 9 22 53 am
---

screen shot 2017-09-20 at 9 21 31 am


Looks like a flow issue. and the `mochi` 💣 . i'll see what's up there

File: src/actions/ui.js [hunk 3]
```diff
@@ -119,8 +120,14 @@ export function clearHighlightLineRange() {
 }
 
 export function toggleConditionalBreakpointPanel(line: null | number) {
```
Comment (bomsy) [inline src/actions/ui.js]: @nyrosmith the flow issue had to do with the line input
This should fix it `export function toggleConditionalBreakpointPanel(line?: number) {`

Comment (nyrosmith) [reply src/actions/ui.js]: Thanks @bomsy for looking at it and helping me out!

**Comment by nyrosmith** [comment]
/remind me to finish this PR tomorrow

**Comment by nyrosmith** [comment]
Closing because of https://github.com/devtools-html/debugger.html/pull/4157

**Pull request closed without merging**